app/routes/queries.listings.tsx

```
@@ -149,14 +188,14 @@ const Index = () => {
           </SmallFormContainer>
-        </div>
-        {listings && listings.listings && listings.listings.length > 0 && (
-          <>
-            {resultTitle && <TitleH2 title={resultTitle} />}
-            <Results data={listings} />
-          </>
-        )}
-      </>
+        </PremiumPaywall>
+      </div>
+      {listings && listings.listings && listings.listings.length > 0 && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)